### PR TITLE
dxSpeedDialAction - Fix component in onClick handler args (T731894)

### PR DESCRIPTION
**Pull request by groshenkovamarina**

### CODE REVIEW

File: js/ui/speed_dial_action/speed_dial_main_item.js [hunk 1]
```diff
@@ -84,6 +84,12 @@ const SpeedDialMainItem = SpeedDialItem.inherit({
         this._setClickAction();
     },
 
+    _defaultActionArgs() {
+        return {
+            component: this.option("actions").length === 1 ? this.option("actions")[0] : this
```
Comment (babich-a) [inline js/ui/speed_dial_action/speed_dial_main_item.js]: I suggest not to call this.option("actions") twice.

File: testing/tests/DevExpress.ui.widgets/floatingActionButton.tests.js [hunk 2]
```diff
@@ -307,3 +305,38 @@ QUnit.module("check action buttons position", (hooks) => {
     });
 });
 
+
+QUnit.module("check action buttons click args", (hooks) => {
```
Comment (babich-a) [inline testing/tests/DevExpress.ui.widgets/floatingActionButton.tests.js:309]: Why do floatingActionButton.tests test dxSpeedDialAction widget?

File: js/ui/speed_dial_action/speed_dial_item.js [hunk 3]
```diff
@@ -81,10 +81,16 @@ const SpeedDialItem = Overlay.inherit({
 
         eventsEngine.off(overlayContent, eventName);
         eventsEngine.on(overlayContent, eventName, (e) => {
-            this._clickAction({ event: e });
+            this._clickAction(extend({ event: e, element: this.$element() }));
```
Comment (babich-a) [inline js/ui/speed_dial_action/speed_dial_item.js]: What 'extend' with one argument should do? All tests passed without it.

**Review by babich-a — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.ui.widgets/floatingActionButton.tests.js (hunk 2 above)
Comment (groshenkovamarina) [reply testing/tests/DevExpress.ui.widgets/floatingActionButton.tests.js:309]: Because dxSpeedDialAction widget - part of  FAB costruction

Comment (babich-a) [reply testing/tests/DevExpress.ui.widgets/floatingActionButton.tests.js:309]: What is 'FAB construstion' mean? We have module 'ui/speed_dial_action' and widget dxSpeedDialAction. It is impossible to find tests for 'speed_dial_action' by file name now. I think we have to have the same names here 'js/ui/speed_dial_action' and here 'testing/tests/DevExpress.ui.widgets/floatingActionButton'.

Comment (groshenkovamarina) [reply testing/tests/DevExpress.ui.widgets/floatingActionButton.tests.js:309]: Ок

**Review by babich-a — APPROVED** [review]

**Pull request merged**